### PR TITLE
chore: invite Billlynch

### PR DESCRIPTION
**Pull request by toddbaert**
@Billlynch has been working on some related JS stuff, and helping with the client spec.

Please comment here if you'd like to join or not. Once this is merged you'll get an invite.

### CODE REVIEW

**Review by beeme1mr — APPROVED** [review]

**Review by beeme1mr — CHANGES_REQUESTED** [review]
Let's wait until Bill approves this PR before we merge it.

**Review by kinyoklion — APPROVED** [review]

**Review by Billlynch — APPROVED** [review]
Thank you! :)

**Review by beeme1mr — APPROVED** [review]

**Pull request merged**
